doc/en/components/grids/_shared/cell-editing.md

```diff
@@ -201,6 +202,114 @@ This code is used in the sample below which implements an [SelectComponent](../s
 
 <!-- end: Angular -->
 
+<!-- Blazor -->
+
+If you want to provide a custom template which will be applied to a cell, you can pass such template either to the cell itself, or to its header. 
+
+```Razor
+
+<IgbColumn
+            Field="race"
+            Header="Race"
+            DataType="GridColumnDataType.String"
+            InlineEditorTemplateScript="WebGridCellEditCellTemplate"
```

Comment:
Prop value "WebGridCellEditCellTemplate" does not match with the one defined in js.

Reply:
Also it could be formatted a bit better:
![image](https://github.com/IgniteUI/igniteui-xplat-docs/assets/10397980/536bcf7b-aa4e-493c-9c17-f610519bd0d9)